internal/testutil/retry.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package testutil
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func Flaky(t *testing.T, max int, sleep time.Duration, f func(r *R)) bool {
```

Comment:
It just sleeps between.

No need for exponential backoff - that's for production code. These are tests.